common/src/main/java/feast/common/logging/AuditLogger.java

```diff
@@ -30,20 +31,23 @@
 import org.slf4j.MarkerFactory;
 import org.slf4j.event.Level;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.info.BuildProperties;
 import org.springframework.stereotype.Component;
 
 @Slf4j
 @Component
 public class AuditLogger {
   private static final Marker AUDIT_MARKER = MarkerFactory.getMarker("AUDIT_MARK");
   private static AuditLogProperties properties;
+  private static FeastInstance instance;
 
   @Autowired
-  public AuditLogger(LoggingProperties loggingProperties) {
+  public AuditLogger(LoggingProperties loggingProperties, BuildProperties buildProperties) {
     // Spring runs this constructor when creating the AuditLogger bean,
     // which allows us to populate the AuditLogger class with dependencies.
     // This allows us to use the dependencies in the AuditLogger's static methods
     AuditLogger.properties = loggingProperties.getAudit();
+    instance = FeastInstance.of(buildProperties.getArtifact(), buildProperties.getVersion());
```

Comment:
Still not sure why we need to persist with FeastInstance.

Reply:
`FeastInstance` consolidates fields common to describing a feast instance and prevents a field explosion in AuditLogEntry sub classes:
https://github.com/feast-dev/feast/blob/c8271ed40e6203009f7fc436b0e13cf52e8cdd7e/common/src/main/java/feast/common/logging/entry/ActionAuditLogEntry.java#L39